Assignment 6/Ques4.js

```diff
@@ -0,0 +1,37 @@
+// Create a class called `Person` that has properties for `firstName`, `lastName`, and `age`. The class should also have a method called `fullName` that returns the person's full name.
+// Additionally, the class should have a method called `averageAge` that takes in an array of `Person` objects and returns the average age of all the people in the array.
+
+
+class Person{
+    firstName;
+    lastName;
+    age;
+    constructor(firstname,lastname,age)
+    {
+        this.firstName=firstname;
+        this.lastName=lastname;
+        this.age=age;
+    }
+    fullName()
+    {
+        return this.firstName+" "+ this.lastName;
+    }
+    static averageAge(array)
+    {
+        let sum=0;
+        let totalpeople=array.length-1;
+        let result=array.reduce((accumulator,current)=>{
+            
+            return accumulator=accumulator+current;
```

Comment:
- You've applied `array.reduce()` on an array of objects, which will store objects in the **result** variable.
- Recheck the method.
- (Hint: Access age from the object and count the total age using the `reduce()` method.)